<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSeccionesAlmacen</class>
 <widget class="QDialog" name="frmSeccionesAlmacen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">background-color: #304163;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Secciones</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">background-color: #304163;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Familias</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QTableView" name="tabla_secciones"/>
   </item>
   <item row="1" column="1">
    <widget class="QTableView" name="tabla_familias"/>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Código:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="txtCodigo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Sección:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QLineEdit" name="txtSeccion"/>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnCambiarImagen">
        <property name="text">
         <string>Cambiar Imagen</string>
        </property>
        <property name="icon">
         <iconset resource="../maya.qrc">
          <normaloff>:/Icons/PNG/refrescar.png</normaloff>:/Icons/PNG/refrescar.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="4" rowspan="2">
       <widget class="QLabel" name="imagen">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../maya.qrc">:/Icons/PNG/Box.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnEliminarImagen">
        <property name="text">
         <string>Eliminar Imagen</string>
        </property>
        <property name="icon">
         <iconset resource="../maya.qrc">
          <normaloff>:/Icons/PNG/borrar.png</normaloff>:/Icons/PNG/borrar.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
       <property name="icon">
        <iconset resource="../maya.qrc">
         <normaloff>:/Icons/PNG/Save.png</normaloff>:/Icons/PNG/Save.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnAnadir">
       <property name="text">
        <string>Añadir</string>
       </property>
       <property name="icon">
        <iconset resource="../maya.qrc">
         <normaloff>:/Icons/PNG/add.png</normaloff>:/Icons/PNG/add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Deshacer</string>
       </property>
       <property name="icon">
        <iconset resource="../maya.qrc">
         <normaloff>:/Icons/PNG/undo.png</normaloff>:/Icons/PNG/undo.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnEditar">
       <property name="text">
        <string>Editar</string>
       </property>
       <property name="icon">
        <iconset resource="../maya.qrc">
         <normaloff>:/Icons/PNG/edit.png</normaloff>:/Icons/PNG/edit.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Eliminar</string>
       </property>
       <property name="icon">
        <iconset resource="../maya.qrc">
         <normaloff>:/Icons/PNG/borrar.png</normaloff>:/Icons/PNG/borrar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabla_secciones</tabstop>
  <tabstop>btnAnadir</tabstop>
  <tabstop>btnEditar</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>txtCodigo</tabstop>
  <tabstop>txtSeccion</tabstop>
  <tabstop>btnCambiarImagen</tabstop>
  <tabstop>btnEliminarImagen</tabstop>
  <tabstop>tabla_familias</tabstop>
 </tabstops>
 <resources>
  <include location="../maya.qrc"/>
 </resources>
 <connections/>
</ui>
